mm2src/mm2_test_helpers/src/for_tests.rs

```diff
@@ -822,11 +822,13 @@ pub fn eth_testnet_conf_trezor() -> Json {
         "coin": "ETH",
         "name": "ethereum",
         "mm2": 1,
-        "chain_id": 1337,
         "max_eth_tx_type": 2,
         "derivation_path": "m/44'/1'", // Trezor uses coin type 1 for testnet
         "protocol": {
-            "type": "ETH"
+            "type": "ETH",
+            "protocol_data": {
+                "chain_id": 1337,
+            }
         },
```

Comment:
for e.g., before this commit, did such a config work?
i see we have no defaults or json mutations to handle such config.

Reply:
Yep, all tests passed, so it worked. Why wouldn't it?